www/js/tripconfirm/trip-confirm-services.js

```diff
@@ -27,24 +27,45 @@ angular.module('emission.tripconfirm.services', ['ionic', "emission.plugin.logge
             });
     }
     
+    var loadAndPopulateOptionsLocales = function (filename) {
+        return $http.get(filename)
+            .then(fillInOptions)
+            .catch(function (err) {
+                // no prompt here since we have a fallback
+                console.log("error "+JSON.stringify(err)+" while reading confirm options, no file for this language, revert to default options");
+                return loadAndPopulateOptions("json/trip_confirm_options.json");
+            });
+    }
     /*
      * Lazily loads the options and returns the chosen one. Using this option
      * instead of an in-memory data structure so that we can return a promise
      * and not have to worry about when the data is available.
      */
     ch.getModeOptions = function() {
         if (!angular.isDefined(ch.modeOptions)) {
-            return loadAndPopulateOptions("json/trip_confirm_options.json")
+            var lang = $translate.use();
+            if (lang != "en") {
+                return loadAndPopulateOptionsLocales("i18n/trip_confirm_options-"+ lang+".json")
+                    .then(function () { return ch.modeOptions; });
+            } else {
+                return loadAndPopulateOptions("json/trip_confirm_options.json")
                 .then(function() { return ch.modeOptions; });
+            }
         } else {
             return Promise.resolve(ch.modeOptions);
         }
     }
 
     ch.getPurposeOptions = function() {
         if (!angular.isDefined(ch.purposeOptions)) {
-            return loadAndPopulateOptions("json/trip_confirm_options.json")
+            var lang = $translate.use();
+            if (lang != "en") {
+                return loadAndPopulateOptionsLocales("i18n/trip_confirm_options-fr.json")
```

Comment:
nit: use `lang` instead of hardcoded `fr`

Reply:
Oups, 5b94703 fixed it !